### PR TITLE
Typos from marvin.cs.uidaho.edu: O

### PR DESCRIPTION
**Pull request by skangas**
This replaces PR #1964. I've added in several variations myself.

See: http://marvin.cs.uidaho.edu/misspell.html

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -25633,16 +25739,25 @@ opulate->populate, opiate, opulent,
 opulates->populates, opiates,
 opyion->option
 opyions->options
+orangatang->orangutan
+orangatangs->orangutans
 orcale->oracle
+orcestrate->orchestrate
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:25745]: Add `orcestra->orchestra` and plural `orcestras->orchestras`?

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -25136,6 +25151,7 @@ obvisous->obvious
 obvisously->obviously
 obyect->object
 obyekt->object
+ocapella->a cappella
```
[the remark this answers is not in the document]
Comment (skangas) [reply codespell_lib/data/dictionary.txt:25154]: Hmm...

>  In referring to singing unaccompanied by instruments, the traditional spelling is the Italian one, a cappella: two words, two Ps, two Ls. The Latin spelling a capella is learned; but in the realm of musical terminology, we usually stick with Italian. The one-word spelling “acapella” is widely used by Americans, including by some performing groups; but this is generally regarded by musical experts as an error. 

https://www.beedictionary.com/common-errors/acapella_vs_a_capella_vs_a_cappella

Maybe these should go in "rare"?

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -25027,8 +25027,19 @@ nuturing->nurturing
 nwe->new
 nwo->now
 o'caml->OCaml
+oaker->ocher
```
[the remark this answers is not in the document]
Comment (skangas) [reply codespell_lib/data/dictionary.txt:25030]: Added.

File: codespell_lib/data/dictionary.txt (hunk 2 above)
[the remark this answers is not in the document]
Comment (DimitriPapadopoulos) [reply codespell_lib/data/dictionary.txt:25154]: Yes, let's try _rare_. We'll remove it if users complain.

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -25198,12 +25217,24 @@ occurrances->occurrences
 occurrencs->occurrences
 occurrs->occurs
 oce->once, one, ore,
+ochreous->ocherous
+ochry->ochery
```
[the remark this answers is not in the document]
Comment (skangas) [reply codespell_lib/data/dictionary.txt:25221]: Another one of these:

```
E   error 'ochreous' should be in aspell dictionaries (en_GB) for dictionary /home/skangas/wip/codespell/codespell_lib/tests/../data/dictionary_en-GB_to_en-US.txt
E   error 'ochrey' should be in aspell dictionaries (en_GB) for dictionary /home/skangas/wip/codespell/codespell_lib/tests/../data/dictionary_en-GB_to_en-US.txt
```

Added to https://github.com/codespell-project/codespell/issues/2657#issuecomment-1365202259 instead.

[the remark this answers is not in the document]
Comment (skangas) [reply codespell_lib/data/dictionary.txt:25221]: The rest is done.

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -25198,12 +25217,24 @@ occurrances->occurrences
...
+ocilator->oscillator
+ocilators->oscillators
 oclock->o'clock
 ocntext->context
 ocorrence->occurrence
 ocorrences->occurrences
 octect->octet
 octects->octets
+octive->octave
+octives->octaves
```
[the remark this answers is not in the document]
Comment (skangas) [reply codespell_lib/data/dictionary.txt:25237]: Yup, added.

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -25316,11 +25371,30 @@ olny->only
 olt->old
 olther->other
 oly->only
+omage->homage
+omages->homages
+omaj->homage
```
[the remark this answers is not in the document]
Comment (skangas) [reply codespell_lib/data/dictionary.txt:25376]: Added.

File: codespell_lib/data/dictionary.txt [hunk 7]
```diff
@@ -25316,11 +25371,30 @@ olny->only
...
+omishints->omniscience
+omisience->omniscience
+omisiences->omnisciences
 omision->omission
 omited->omitted
 omiting->omitting
 omitt->omit
+omlet->omelet
+omlets->omelets
 omlette->omelette
```
[the remark this answers is not in the document]
Comment (skangas) [reply codespell_lib/data/dictionary.txt:25391]: I usually add variations like this when I spot them, even if they're not new. So I added this one too.

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (skangas) [reply codespell_lib/data/dictionary.txt:25745]: Yes. Added.

File: codespell_lib/data/dictionary.txt (hunk 2 above)
[the remark this answers is not in the document]
Comment (skangas) [reply codespell_lib/data/dictionary.txt:25154]: I put it in `rare` for now.

**Review by DimitriPapadopoulos — APPROVED** [review]

**Pull request merged**